Bio/Align/stockholm.py

```diff
@@ -592,15 +602,15 @@ def _format_record(width, start, aligned_sequence, record):
         indices.reverse()
         name = record.id.ljust(width)
         for key, value in record.letter_annotations.items():
-            feature = AlignmentWriter.gr_mapping[key]
+            feature = AlignmentWriter.gr_mapping.get(key, key)
             j = 0
             values = bytearray(b"." * len(aligned_sequence))
             for i, letter in enumerate(aligned_sequence):
                 if letter not in ".-":
                     values[i] = ord(value[j])
                     j += 1
             value = values.decode()
-            line = f"#=GR {name}  {feature}".ljust(start) + value + "\n"
+            line = f"#=GR {name}  {feature} ".ljust(start) + value + "\n"
```

Comment:
Why the extra space after {feature}?

Reply:
Same as above, but here it is more important, because the identifier (name) might be longish, too.

Reply:
OK